tests/ci/rerun_helper.py

```diff
@@ -36,3 +36,9 @@ def is_already_finished_by_status(self):
             ):
                 return True
         return False
+
+    def get_finished_status(self):
+        for status in self.statuses:
+            if self.check_name in status.context:
+                return status.state
```

Comment:
I just found it. It's `get_finished_status`, but you return the state of it. Let's return the status, it will be much more useful. I already know where I'll use it!